src/StorageSync/StorageSync/help/Invoke-AzStorageSyncChangeDetection.md

```diff
@@ -1,6 +1,6 @@
 ---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.StorageSync.dll-Help.xml
-Module Name: Az.StorageSync
+Module Name: Microsoft.Azure.PowerShell.Cmdlets.StorageSync
```

Comment:
Why did this change? Did it come as a result of running the script that regenerates help content?

Reply:
It must have, I didn't edit this file manually. Let me try generating it again

Reply:
I think I somehow generated the help with the wrong Accounts module, it should be fixed now